addons/copy-reporter/addon.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Copy reporters",
```

Comment:
```suggestion
  "name": "Copy reporter text",
```
Can someone come up with abetter name than that?

Reply:
I think values is best for consistency. Let me know if you don't like this though.